src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,19 +34,24 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={query}
+        onChange={(event) => onSelectQuery(event.target.value)}
       />
       <span className="icon is-left">
         <i className="fas fa-magnifying-glass" />
       </span>
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
-        />
-      </span>
+      {query && (
+        <span className="icon is-right" style={{ pointerEvents: 'all' }}>
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
why do you need it?

Reply:
I need to show this button only if the query is set
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/130338460/aac2e2d6-255c-4386-b871-6893ce1688ae)